api/openapi-spec/swagger.json

```diff
@@ -10725,6 +10729,18 @@
      }
     }
    },
+   "v1.NUMA": {
+    "type": "object",
+    "required": [
+     "numaTopologyStrategy"
+    ],
+    "properties": {
+     "numaTopologyStrategy": {
+      "description": "NumaTopologyStrategy defines how to model the numa topology in a guest. Supported vaules are: \"passthrough\"",
```

Comment:
can you imagine other strategies? (I failed to)

Reply:
late idea: `Uniform` which is equivalent to the behavior before this PR?

Reply:
Do you mean `Uniform` as the strategy name?

Yes there are definitely other strategies possible: For instance full manual configuration (not yet possible in k8s but may come) and ,ore migration-conformant toplogies which are mappable easier between servers and still give performance benefits.

Reply:
I see some sub-configuration use-cases on the horizon too. I therefore removed the stretegy field again and gave this strategy its on substruct. See the updated PR description for details and the reasoning.

Reply:
+1 on the new struct approach. That's even further future proofing ourselves.

> can you imagine other strategies? (I failed to)

@dankenigsberg This is a lesson learned from the evolution of `vm.Spec.Running` to `RunStrategy`. Unless we're 100% sure a value is a boolean, we use an enum or struct now. There's enough unknown within the realm of numa scheduling and assignment that we're making this API as flexible as possible in order to serve us in whatever direction this goes in the future.